samples/trumpify/trumpify.py

```diff
@@ -0,0 +1,108 @@
+import os
+import sys
+import threading
+import time
+
+sys.path.append(os.path.abspath('../../'))
+
+from samples.trumpify.twitter_client import TwitterClient
+from daemo.client import DaemoClient
+
+CREDENTIALS_FILE = 'credentials.json'
+
+PROJECT_ID = -1
+
+TW_CONSUMER_KEY = ''
+TW_CONSUMER_SECRET = ''
+TW_ACCESS_TOKEN = ''
+TW_ACCESS_TOKEN_SECRET = ''
+
+INPUT_TWITTER_NAME = 'HillaryClinton'
+MINUTES = 60
+FETCH_INTERVAL_MIN = 5
+TWEET_COUNT = 10
+MONITOR_INTERVAL_MIN = 60
+
+twitter = TwitterClient(
+    consumer_key=TW_CONSUMER_KEY,
+    consumer_secret=TW_CONSUMER_SECRET,
+    token=TW_ACCESS_TOKEN,
+    token_secret=TW_ACCESS_TOKEN_SECRET
+)
+client = DaemoClient(CREDENTIALS_FILE)
+
+
+def fetch_new_tweets(count, interval):
+    while True:
+        messages = twitter.fetch_tweets(twitter_name=INPUT_TWITTER_NAME, count=count, interval=interval)
+
+        if len(messages) > 0:
+            for message in messages:
+                post_to_daemo(message)
+        else:
+            print "@%s has not tweeted in the last %d minutes." % (INPUT_TWITTER_NAME, interval)
+
+        time.sleep(interval)
+
+
+def post_to_daemo(message):
+    text = message.get('text')
+    id = message.get('id')
+
+    client.publish(project_id=PROJECT_ID, tasks=[{
+        "id": id,
+        "tweet": text
+    }], approve=approve_tweet, completed=post_to_twitter, stream=True)
```

Comment:
Right, which is why I suggest not including the stream parameter, since
this is supposed to be as basic an example as possible.

On Tue, Jul 19, 2016 at 9:52 PM, Shirish Goyal <notifications@github.com>
wrote:

> In samples/trumpify/trumpify.py
> <https://github.com/crowdresearch/daemo-api-client/pull/1#discussion_r71465084>
> :
>
> > +            for message in messages:
> > +                post_to_daemo(message)
> > +        else:
> > +            print "@%s has not tweeted in the last %d minutes." % (INPUT_TWITTER_NAME, interval)
> > +
> > +        time.sleep(interval)
> > +
> > +
> > +def post_to_daemo(message):
> > +    text = message.get('text')
> > +    id = message.get('id')
> > +
> > +    client.publish(project_id=PROJECT_ID, tasks=[{
> > +        "id": id,
> > +        "tweet": text
> > +    }], approve=approve_tweet, completed=post_to_twitter, stream=True)
>
> In this example, it doesn't matter as each task is pushed individually so
> it will behave the same.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/crowdresearch/daemo-api-client/pull/1/files/113785338fd5eed4efe15995fda764a984cc1b9f#r71465084>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAPNxZnlLhchbXnjBBrx1fBTx4kPqBCCks5qXalzgaJpZM4I_odP>
> .
>